src/js/modules/setup/controllers/wallet-init/wallet-init.ctrl.js

```diff
@@ -0,0 +1,312 @@
+                    return $scope.sdk.initWallet({identifier: $scope.setupInfo.identifier, password: $scope.setupInfo.password});
+                })
+                .then(function(wallet) {
+                    $analytics.eventTrack("initWallet", {category: "Events"});
+
+                    // time to upgrade to V3 ppl!
+                    if (wallet.walletVersion != blocktrailSDK.Wallet.WALLET_VERSION_V3) {
+                        $scope.updateProgress({title: "UPGRADING_WALLET", body: "UPGRADING_WALLET_BODY"});
+
+                        return wallet.upgradeToV3($scope.setupInfo.password)
+                            .progress(function(progress) {
+                                /*
+                                 * per step we increment the progress bar and display some new progress text
+                                 * some of the text doesn't really match what is being done,
+                                 * but we just want the user to feel like something is happening.
+                                 */
+                                switch (progress) {
+                                    case blocktrailSDK.CREATE_WALLET_PROGRESS_START:
+                                        $scope.updateProgress({title: "UPGRADING_WALLET", body: "UPGRADING_WALLET_BODY"});
+                                        break;
+                                    case blocktrailSDK.CREATE_WALLET_PROGRESS_ENCRYPT_SECRET:
+                                        $scope.updateProgress({title: "UPGRADING_WALLET", body: "CREATING_GENERATE_PRIMARYKEY"});
+                                        break;
+                                    case blocktrailSDK.CREATE_WALLET_PROGRESS_ENCRYPT_PRIMARY:
+                                        $scope.updateProgress({title: "UPGRADING_WALLET", body: "CREATING_GENERATE_BACKUPKEY"});
+                                        break;
+                                    case blocktrailSDK.CREATE_WALLET_PROGRESS_ENCRYPT_RECOVERY:
+                                        $scope.updateProgress({title: "UPGRADING_WALLET", body: "CREATING_GENERATE_RECOVERY"});
+                                        break;
+                                }
+
+                            })
+                            .then(function() {
+                                $scope.updateProgress({title: "UPGRADING_WALLET", body: "UPGRADING_WALLET_BODY"});
+
+                                // bump progress
+                                $scope.progressWidth = 90;
+
+                                return wallet;
+                            });
+
+                    } else {
+                        $log.debug("wallet initialised", wallet);
+                        // bump progress
+                        $scope.progressWidth = 90;
+                        return wallet;
+                    }
+                }, function(error) {
+                    if (error.message.match(/not found/) || error.message.match(/couldn"t be found/)) {
```

Comment:
should be `if (error.message.match(/not found/) || error.message.match(/couldn't be found/)) {`

somehow the `couldn't` was replaced with `couldn"t`